test/Land/Model/haverkamp_implicit_test.jl

```diff
@@ -0,0 +1,200 @@
+
+    soil_param_functions = SoilParamFunctions{FT}(
+        porosity = 0.495,
+        Ksat = 0.0443 / (3600 * 100),
+        S_s = 1e-3,
+    )
+    # Keep in mind that what is passed is aux⁻
+    # Fluxes are multiplied by ẑ (normal to the surface, -normal to the bottom,
+    # where normal point outs of the domain.)
+    surface_value = FT(0.494)
+    bottom_flux_multiplier = FT(1.0)
+    initial_moisture = FT(0.24)
+
+    surface_state = (aux, t) -> surface_value
+    bottom_flux = (aux, t) -> aux.soil.water.K * bottom_flux_multiplier
+    ϑ_l0 = (aux) -> initial_moisture
+
+    soil_water_model = SoilWaterModel(
+        FT;
+        moisture_factor = MoistureDependent{FT}(),
+        hydraulics = Haverkamp{FT}(),
+        initialϑ_l = ϑ_l0,
+        dirichlet_bc = Dirichlet(
+            surface_state = surface_state,
+            bottom_state = nothing,
+        ),
+        neumann_bc = Neumann(surface_flux = nothing, bottom_flux = bottom_flux),
+    )
+
+    m_soil = SoilModel(soil_param_functions, soil_water_model, soil_heat_model)
+    sources = ()
+    m = LandModel(
+        param_set,
+        m_soil;
+        source = sources,
+        init_state_prognostic = init_soil_water!,
+    )
+
+
+    N_poly = 5
+    nelem_vert = 10
+
+    # Specify the domain boundaries
+    zmax = FT(0)
+    zmin = FT(-1)
+
+    driver_config = ClimateMachine.SingleStackConfiguration(
+        "LandModel",
+        N_poly,
+        nelem_vert,
+        zmax,
+        param_set,
+        m;
+        zmin = zmin,
+        numerical_flux_first_order = CentralNumericalFluxFirstOrder(),
+    )
+
+    t0 = FT(0)
+    timeend = FT(60 * 60 * 24)
+
+    dt = FT(200)
+
+    solver_config = ClimateMachine.SolverConfiguration(
+        t0,
+        timeend,
+        driver_config,
+        ode_dt = dt,
+    )
+
+    #################### Change the ode_solver
+
+    dg = solver_config.dg
+    Q = solver_config.Q
+
+    vdg = DGModel(
+        driver_config.bl,
+        driver_config.grid,
+        driver_config.numerical_flux_first_order,
+        driver_config.numerical_flux_second_order,
+        driver_config.numerical_flux_gradient,
+        state_auxiliary = dg.state_auxiliary,
+        direction = VerticalDirection(),
+    )
+
+
+
+    linearsolver = BatchedGeneralizedMinimalResidual(
+        dg,
+        Q;
+        max_subspace_size = 30,
+        atol = -1.0,
+        rtol = 1e-9,
+    )
+
+    """
+    N(q)(Q) = Qhat  => F(Q) = N(q)(Q) - Qhat
+
+    F(Q) == 0
+    ||F(Q^i) || / ||F(Q^0) || < tol
+
+    """
+    nonlinearsolver =
+        JacobianFreeNewtonKrylovSolver(Q, linearsolver; tol = 1e-9)
+
+    ode_solver = ARK548L2SA2KennedyCarpenter(
+        dg,
+        vdg,
+        NonLinearBackwardEulerSolver(
+            nonlinearsolver;
+            isadjustable = true,
+            preconditioner_update_freq = 100,
+        ),
+        Q;
+        dt = dt,
+        t0 = 0,
+        split_explicit_implicit = false,
```

Comment:
I thought vdg was tendency the implicit solver was used with, and dg-vdg would use the explicit solver. Is that true in general, but here, since we only have one column, you can set split_explicit_implicit = false?

Reply:
" I thought vdg was tendency the implicit solver was used with, and dg-vdg would use the explicit solver. Is that true in general, but here, since we only have one column ", yes
my understanding of "split_explicit_implicit = false",  we use dg-vdg as explicit part instead of defining it https://clima.github.io/ClimateMachine.jl/latest/APIs/Numerics/ODESolvers/ODESolvers/#ClimateMachine.ODESolvers.AdditiveRungeKutta